unfoldingWord/translationCore
# Now Copying translationHelps from Static folder to User Resources folder

## This pull request addresses:
Now Copying translationHelps from Static folder to User Resources folder

## How to test this pull request:
- Make sure you delete the bibles from the user resources folder and then open tC.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1869)
